src/main/resources/db/changelog/changes/07-create_shopping_carts_table.yaml

```diff
@@ -0,0 +1,28 @@
+databaseChangeLog:
+  - changeSet:
+      id: create_shopping_carts_table
+      author: Volodymyr Ozemblovskyi
+      changes:
+        - createTable:
+            tableName: shopping_carts
+            columns:
+              - column:
+                  name: id
+                  type: bigint
+                  autoIncrement: true
+                  constraints:
+                    primaryKey: true
+                    nullable: false
```

Comment:
nullable is redundant here as you have primary key constraint, but that's a minor thing